content/en/tracing/trace_collection/trace_context_propagation/_index.md

```diff
@@ -118,13 +118,14 @@ In addition to the environment variable configuration, you can also update the p
 
 The Datadog Python SDK supports the following trace context formats, including deprecated configuration values:
 
-| Format                 | Configuration Value |
-|------------------------|---------------------|
-| [Datadog][1]           | `datadog`           |
-| [W3C Trace Context][2] | `tracecontext`      |
-| [B3 Single][3]         | `b3`                |
-| [B3 Multi][4]          | `b3multi`           |
-| [None][5]              | `none`              |
+| Format                 | Configuration Value             |
+|------------------------|---------------------------------|
+| [Datadog][1]           | `datadog`                       |
+| [W3C Trace Context][2] | `tracecontext`                  |
+| [B3 Single][3]         | `b3`                            |
+|                        | `b3 single header` (deprecated) |
```

Comment:
I saw `_Language Dependent Value_` [above](https://github.com/DataDog/documentation/pull/26351/files#diff-ec5fd32d7c8381323dba6d07d57491c42132c89cd9ec214eb08cb6a7954f42a1R72) and thought this was supported everywhere. Apparently Ruby doesn't support it, but I'm awfully tempted to standardize on it...